wasm/jit/jit_arm64_test.go

```diff
@@ -351,8 +351,11 @@ func TestArm64Compiler_loadValueOnStackToRegister(t *testing.T) {
 	}
 }
 
-func TestArm64Compiler_compile_Le_Lt_Gt_Ge(t *testing.T) {
+func TestArm64Compiler_compile_Le_Lt_Gt_Ge_Eq_Eqz_Ne(t *testing.T) {
```

Comment:
```suggestion
// TODO: break this up somehow so that the test name is more readable
func TestArm64Compiler_compile_Le_Lt_Gt_Ge_Eq_Eqz_Ne(t *testing.T) {
```